openapi/openapi/src/main/scala/endpoints/documented/openapi/Requests.scala

```diff
@@ -2,6 +2,8 @@ package endpoints
 package documented
 package openapi
 
+import endpoints.documented.openapi.model.MediaType
```

Comment:
Why this import is needed?

Reply:
It is used [here](https://github.com/julienrf/endpoints/blob/0cf6d19045b13eea911670d64ae96bf5b93fc48f/openapi/openapi/src/main/scala/endpoints/documented/openapi/Requests.scala#L38). I moved the OpenAPI ADT into a `model` subpackage.